src/views/Player.vue

```diff
@@ -376,13 +393,20 @@ export default {
 
     async setPlayerProperties(player) {
       player.on("ready", () => {
+        this.player.currentTime = Math.abs(this.previousPlayerTime - 5)
+
```

Comment:
Start playing from where the user left in the previous session.
TODO:
If the video is finished, or the user has reached the end, start playing from beginning